### DISCUSSION TITLE
Regarding Data collection from Income tax portal

### DISCUSSION THREAD

**Discussion by kamatchi11** [madrecha/jamku#1124] [General]
i just happened to see Jamku app, it looks good and seems very useful, i would like to understand few details - 
1. how data sync happen between IT portal and Jamku, is that happen thru some API? 
2. Where do you save the data/downloaded pdf file ?

**Comment by adarshmadrecha** [comment]
@kamatchi11 
1. Currently we don't have integration with Jamku Office Management Software and Jamku Portal (IT, GST, Traces).
We are working on the same. Hopefully, by early next year it will be integrated.

2. The pdf files are stored in AWS Mumbai data center.
